dll/cpl/appwiz/lang/ro-RO.rc

```diff
@@ -88,8 +89,8 @@ BEGIN
     IDS_DWL_FAILED "Descărcarea pachetului Gecko a eșuat. Asigurați-vă că aveți conexiune la internet pentru a putea descărca. Asistentul va continua fără a instala pachetul Gecko."
     IDS_CANTMAKEINETSHORTCUT "Eșec în crearea scurtăturii la Internet."
     IDS_CANTMAKESHORTCUT "Eșec în crearea scurtăturii."
-    IDS_NO_MEMORY "No memory could be allocated!"
-    IDS_NO_DIRECTORY "No directory given!"
-    IDS_INVALID_PATH "The given path is invalid!"
-    IDS_INVALID_NAME "The shortcut name you entered either contained characters that are invalid for file names or was too long."
+    IDS_NO_MEMORY "Nu s-a putut aloca memorie!"
```

Comment:
Consider using the passive voice (rom. diateza pasivă) for such cases (see the 15th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)).
I.e. „Nu a putut fi alocată memorie!”